### ISSUE TITLE
[zandronum linux] zdoom-dhtp-20180514.pk3 not loading

### ISSUE THREAD

**Issue by escrevebastante** [KuriKai/DHTP#40]
I followed the instructions for the installation and changed the .ini file to include the .pk3 file but looks like they are not loaded on the game. Through the console I can see them being loaded : 

adding /home/user/.config/zandronum/zdoom-dhtp-20180514.pk3, 762 lumps

But in the game the textures simply don't appear. I tried also with opengl with no change. 
Other textures like NAHDSPv0.2.pk3 are being loaded fine... What am I missing?

**Comment by escrevebastante** [comment]
Actually i also tried with zandronum on windows. Same thing

**Comment by AcidV** [comment]
With some new updates the pack doesn't work for Zandronum anymore. 

2 ways to fix: Download an older version from https://www.wad-archive.com (use search function)

Or you can manually edit the zdoom dhtp .pk3 file with WinRAR and rename the folder "filter"  to "hires" then install normally, it should work.

**Comment by AcidV** [comment]
You will to put the modified .pk3 in the announcers folder for it to load. Or you can manually add it to your wads list if you wish.

**Comment by KuriKai** [comment]
so it should not be "filter/doom/hires" but should be "hires/doom/hires"?

**Comment by KuriKai** [comment]
Is this still an issue?

Latest release is packed "filter/doom/hires"

**Comment by nuku97** [comment]
Yes, the file zdoom-dhtp-20180514.pk3 still does not work with Zandronum. Following the instructions from AcidV (renaming the folder "filter" to "hires") however works.

**Comment by nuku97** [comment]
I looked into this again and actually it is not sufficient to just rename the "filter" to "hires". The problem is that Zandronum does not support lump filtering (https://zdoom.org/wiki/Lump_filtering) that allows loading of lumps depending on the game or gametype like GZDoom. 

The only way which I found to get this working correctly with Zandronum was to split the single PK3-file into separate PK3 files and load them according to the game and in the right order.

I created the separate PK3-Files like this:

```
git clone https://github.com/KuriKai/DHTP.git

mkdir DHTP/splitpack 

cd DHTP/splitpack

for filename in "../docs/"*; do base_name=$(basename $filename); cat "../README.md" "$filename" "../credits.txt" > "README_$base_name"; done

mv ../textures/doom1 hires && zip -r DHTP-Addon-Doom1.pk3 . -i \*.PNG \*.png \*.txt && rm -Rf hires

mv ../textures/doom2 hires && zip -r DHTP-Addon-Doom2.pk3 . -i \*.PNG \*.png \*.txt && rm -Rf hires

mv ../textures/doom2-plut hires && zip -r DHTP-Addon-Plutonia.pk3 . -i \*.PNG \*.png \*.txt && rm -Rf hires

mv ../textures/doom2-tnt hires && zip -r DHTP-Addon-TNT.pk3 . -i \*.PNG \*.png \*.txt && rm -Rf hires

mv ../textures hires && mv ../flats/* hires/. && zip -r DHTP-Base.pk3 . -i \*.PNG \*.png \*.txt && rm -Rf hires README_*

```

This creates the following 5 files:

- DHTP-Base.pk3 - Base pack to be used with every game
- DHTP-Addon-Doom1.pk3 - Doom1 specific files to be loaded in addition to DHTP-Base.pk3 for Doom1
- DHTP-Addon-Doom2.pk3 - Doom2 specific files to be loaded in addition to DHTP-Base.pk3 for Doom2
- DHTP-Addon-Plutonia.pk3 - Plutonia specific files to be loaded in addition to DHTP-Base.pk3 and DHTP-Addon-Doom2.pk3 for Plutonia
- DHTP-Addon-TNT.pk3 - TNT specific files to be loaded in addition to DHTP-Base.pk3 and DHTP-Addon-Doom2.pk3 for TNT

I would start the game than as follows, for example:

Doom1:
`zandronum -iwad DOOM.WAD -file DHTP-Base.pk3 DHTP-Addon-Doom1.pk3 
`

Plutonia:
`zandronum -iwad PLUTONIA.WAD -file DHTP-Base.pk3 DHTP-Addon-Doom2.pk3 DHTP-Addon-Plutonia.pk3
`

Note: The separate files do not only work with Zandronum, but also with GZDoom. Of course less convenient than just a single file with lump filtering.

**Comment by Morgan-6Freedom** [comment]
@nuku97 do you mind uploading those pk3 files ? i'm not on linux and cannot create them
